### DISCUSSION TITLE
如何设置更多breakpoints，例如3xl...

### DISCUSSION THREAD

**Discussion by xmsz-stu** [unocss/unocss#3243] [Q&A]
背景
- 我想要配置3xl:1600 ...

**Comment by zyyv** [comment]
https://unocss.dev/config/theme#breakpoints

**Reply by xmsz-stu** [reply]
我这了 但是3xl不能生效 只能生效 sm md lg xl ，就是只能生效unocss里原来有的，不能新增

**Reply by zyyv** [reply]
那你可以去看看我们的测试用例

**Reply by xmsz-stu** [reply]
https://unocss.dev/play/?html=DwEwlgbgBAxgNgQwM5ILwCIC2IBcAXAUwA88BaAJwJCiU32LICNEYBrKAZiLnpNOYCuBdAD4AUFCiAcjMBOQWOAB6cBBFA&config=PTAEFMGcBsEsDsAuBaAJrSBDARtcz5wAPFOQ0eAe2QCcpEbYBjRcVZWAWwAdKbFIAKC69%2BoAN6DQoVOABmCcAGFK8BQHMANFNDc6kcIgCCiBrGwBXRLDkBPbdL1RDASSarID3fsMBVKtoAvqByNJScoADkFlRMkJCRgoLEoogy8pgW0GmyCoQqarDqABSS0ogAFuCc4ABcEjrS2HSYANa8CAL1ZdKgkJz1kQCMAGwADNxEkV7SnKiDAMwATBNTM6AARAtE0Bv1GwAsAByrGzqBXhc6TgZdoADajd7OiP6UxQCUXgC6QR%2BCQA&css=PQKgBA6gTglgLgUzAYwK4Gc4HsC2YDCAyoWABYJQIA0YAhgHYAmYcUD6AZllDhWOqgAOg7nAB0YAGLcwCAB60cggDYIAXGBDAAUKDBi0mXGADe2sGC704AWgDuCGAHNScDQFYADJ4Dc5sAACtMLKAJ5gggCMLPK2ABR2pPBIcsoAlH4WAEa0yADWTlBYqEw2yFjK3Bpw5LxxAOTllVDoYpSMYgAs3vUZ2gC%2BmsBAA&options=N4IgLgTghgdgzgMwPYQLYAkyoDYgFwJTZwCmAvkA

用playground确实可以...  我等会在项目再试试

**Reply by jie-Yoy** [reply]
我这边也遇到这个问题，但是经过测试用例发现，如果集成第三方预设样式，配置断点3xl就不会生效。应该是编译顺序的问题，就算我调整引入顺序编译结果也是一样。自定义的breakpoints会在最上面，会被引入的预设覆盖掉，导致失效。应该是 css 的响应式执行顺序有关。
然后我想问下，如果要修改媒体查询，改为 @media (max-width: 1920px) {} 这样应该怎么进行配置。

**Reply by zyyv** [reply]
我觉得没有任何问题，应该是你自己配置的问题。如果你要寻求帮助，请提供最小复现，不然没人知道你是什么问题，谢谢/
